lib/cancan/model_adapters/active_record_5_adapter.rb

```diff
@@ -0,0 +1,58 @@
+module CanCan
+  module ModelAdapters
+    class ActiveRecord5Adapter < AbstractAdapter
```

Comment:
They are almost identical apart from the version checks with the exception of the sanitize_sql method which has a slightly different structure around the predicate builder. To be honest it was a few months ago that I looked at this originally and I can't recall the exact reasons for those changes.